aws/resource_aws_msk_scram_secret.go

```diff
@@ -0,0 +1,216 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kafka"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsMskScramSecret() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsMskScramSecretCreate,
+		Read:   resourceAwsMskScramSecretRead,
+		Update: resourceAwsMskScramSecretUpdate,
+		Delete: resourceAwsMskScramSecretDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"cluster_arn": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
let's add a validation for this ARN e.g.
```suggestion
				Required: true,
				ValidateFunc: validateArn,
```

Reply:
should this attribute change, should this force a new resource?